example/benchmark/main.go

```diff
@@ -41,7 +41,7 @@ func main() {
 func task(ctx context.Context) (streams.Task, error) {
 	builder := streams.NewStreamBuilder()
 	builder.Source("nil-source", newNilSource(ctx)).
-		Map("do-nothing", nothingMapper)
+		Map("do-nothing", streams.MapperFunc(nothingMapper))
```

Comment:
I didn't want to modify the existing examples code before you approved those map functions (to avoid wasted effort). I will do it now, though, that you have accepted the idea.